File: microbench/node/array_sum.js
function buildAndSum(size) {
  const list = [];
  for (let i = 0; i < size; i++) {
    list.push(i);
  }

  let total = 0;
  for (const item of list) {
    total += item;
  }
  return total;
}

const result = buildAndSum(1000000);
console.log(result);

File: microbench/python/array_sum.py
def build_and_sum(size):
    lst = []
    for i in range(size):
        lst.append(i)

    total = 0
    for item in lst:
        total += item
    return total


if __name__ == "__main__":
    result = build_and_sum(1000000)
    print(result)
